libmscore/textbase.cpp

```diff
@@ -2563,7 +2563,9 @@ void TextBase::draw(QPainter* p) const
       if (hasFrame()) {
             if (frameWidth().val() != 0.0) {
                   QColor fColor = curColor(visible(), frameColor());
-                  QPen pen(fColor, frameWidth().val() * spatium(), Qt::SolidLine,
+                  qreal frameWidthVal = frameWidth().val() * (sizeIsSpatiumDependent() ? spatium() : 1);
```

Comment:
frameWidth() is a Spatium property. As such, its value must always be multiplied by spatium(), even if the text size is not spatium dependent. Unless you are using the precomputed value, which you are not.

Reply:
My reasoning here was that if you've got text that you want to ignore spatium settings, surely you'd want the text frame, if any, to also obey this setting? The frame is part of the text, not a separate element, and I would see it as expected behaviour that if the text size doesn't depend on spatium, neither should the frame.

Reply:
Then at least multiply frameWidth().val() by (spatium() / baseSpatium). As it is, if the text size is not spatium dependent, the frame is so faint it is barely noticeable at the default frame width.

Reply:
Wait, no, that won't work. It would cancel out.

Reply:
Ignore my earlier comment. I guess what I really meant was if the text size is not spatium dependent, frameWidth().val() should be multiplied by baseSpatium.

Reply:
Oh yeah, of course. My mistake there. I'll submit a follow up PR later.